src/docs/references/extended-data/index.md

```diff
@@ -86,12 +79,13 @@ schema is provided for a given extended data key, the API can use this
 information to make querying the extended data possible via some API
 endpoints. For instance
 [/listings/query](https://www.sharetribe.com/api-reference/marketplace.html#query-listings)
-supports querying listings by public data.
+supports querying listings by public data or metadata.
 
 Search schemas can be managed by
 [Flex CLI](/flex-cli/getting-started-with-flex-cli/). With the CLI, you
 can list, set and unset search schemas to listing's Public data and
-Public metadata. The commands to manage search schemas are:
+Metadata as well as user's Metadata, Private, Protected and Public.
```

Comment:
```suggestion
metadata as well as user's metadata, private, protected and public data.
```

Reply:
@bladealslayer Original documentation used capitalized words, any reason to change it to lower-cased?